internal/yggdrasil/yggdrasil.go

```diff
@@ -446,9 +446,16 @@ func (h *Handler) toWorkloadList(ctx context.Context, logger logr.Logger, deploy
 			data = &models.DataConfiguration{Paths: paths}
 		}
 
+		podmanLabels := getPodmanLabels(deployment.Labels)
+		if !validatePodmanLabels(podmanLabels, deployment) {
+			msg := "No volume with authfile path was found, please check if you mount the auth file."
+			h.recorder.Event(device, corev1.EventTypeWarning, "Misconfiguration", msg)
```

Comment:
Is there an option to control the frequency of this event to prevent it from flooding (every 15 seconds)

Reply:
A logic for this would need to be introduced, there is nothing bultin. But I am thinking that such an event is not important for the user in the device Events IMHO, so I think it's ok, to be removed.